src/service_manager.py

```diff
@@ -173,7 +184,11 @@ async def _create_view(
         print("Creating view ... ", end="", flush=True)
         try:
             await store.create_search_view(
-                view_name, db_collection_name, view_spec, analyzers.get_analyzer)
+                view_name,
+                db_collection_name,
+                view_spec,
+                analyzers.get_analyzer,
+                include_all_fields=create_generic_view)
             print(f"done in {time.time() - t0:.2f} seconds.")
             return view_name
         except ViewExistsError:
```

Comment:
The prints below this point do apply for generic specs

Reply:
👍

Reply:
: